index2.js

```diff
@@ -0,0 +1,15 @@
+
+const sumFunc =(arr)=>{
+    if (!Array.isArray(arr)) return;
+    return arr.map((item,i,currentAr)=>{ 
```

Comment:
don't use short names of variables. `currentArray` and `index` looks rather clear